praw/models/reddit/redditor.py

```diff
@@ -203,11 +203,11 @@ def block(self):
     def distrust(self):
         """Remove the Redditor from your whitelist of trusted users.
 
-        For example, to remove Redditor ``spez`` from your whitelist:
+        For example, to remove Redditor ``jc7254`` from your whitelist:
 
         .. code-block:: python
 
-            reddit.redditor("spez").distrust()
+            reddit.redditor("jc7264").distrust()
```

Comment:
Sort of.  When I made the initial PR I just picked names of early Reddit employees. In hindsight, it's not such a good idea to have the CEO as someone to "distrust". So I replaced his name with Jose Canseco's, a baseball player who did a famously bad  [AMA](https://old.reddit.com/r/IAmA/comments/1clw9o/i_am_jose_canseco_famed_steroid_user_and_former/) years ago. I think [this comment](https://old.reddit.com/r/IAmA/comments/1clw9o/i_am_jose_canseco_famed_steroid_user_and_former/c9hs7hr/) is a good example of why his account is a good fit.

Reply:
I see and I agree that it could be interpreted negatively. I think we should come up with an example username and use it globally. 

For now, let's spilt this to it's own PR. I'd actually like to use a username that is suspended for the distrust method specifically. There are spam/bad actor accounts out there that are suspended but preserved for public reference.

In the meantime, I'll poke @bboe and @jarhill0 for their thoughts on using a set example username, subreddit, submission, etc. to be used in docs because of the connotations￼ some methods could convey. I think we've briefly discussed it but nothing was decided.

Reply:
> I see and I agree that it could be interpreted negatively. I think we should come up with an example username and use it globally.
> 
> For now, let's spilt this to it's own PR. I'd actually like to use a username that is suspended for the distrust method specifically. There are spam/bad actor accounts out there that are suspended but preserved for public reference.
> 
> In the meantime, I'll poke @bboe and @jarhill0 for their thoughts on using a set example username, subreddit, submission, etc. to be used in docs because of the connotations￼ some methods could convey. I think we've briefly discussed it but nothing was decided.

I can take it out. For what it's worth, the example for block also uses spez. Blocking employees or removing them from your whitelist doesn't actually do anything, since they can send you a message regardless of what your settings are.

Reply:
True, which is another reason we should find a good example username to use in examples. Having an employee as the target for the `block` example implies that one could successfully block an reddit employee which isn't true.

Reply:
This is an interesting discussion about usernames. I personally am impartial about the names in the documentation so long as anyone wouldn't be annoyed by people running said examples (e.g., I do not want my account to be messaged by people testing). IMO Using the co-creator of Reddit I think is reasonable, nevertheless, I support whatever @LilSpazJoekp recommends.